docs/api/creating-api.md

```diff
@@ -0,0 +1,115 @@
+# Creating the API
+
+## Creating the API on the OneBlink Console
+
+Firstly, we will need to make sure we have something to deploy to when we are ready. Let us go onto the [OneBlink Console](console.oneblink.io). Once we are in our account, go to the "API Hosting" side tab under the Advanced heading. Once here, let us click the plus button in the bottom right hand corner.
+
+![An image showing the API Hosting Page highlighting the button that needs to be clicked"](../pics/APIHostingPointingArrow.png)
+
+It's time to fill out some information. We only need to fill in the API subdomain, so call the subdomain what you would like. In my example, I will call it "example".
```

Comment:
Might want to use an example that is tangible for customers rather than something abstract

Reply:
Would it be good to have something like 'my-api' or 'my-project'? Or write about how you should name the API based on the work you are doing for it?

Reply:
I'd try to think of a real world example like `audits`